### ISSUE TITLE
LIPIS-based content loss is missing

### ISSUE THREAD

**Issue by Kaushalya** [04RR/MedSRGAN#3]
The paper claims to use a content loss based on a pre-trained VGG-19 model. However, only an L1-loss is used in the training code. How was the performance mentioned in the paper achieved? Is it with L1 loss? 

![image](https://user-images.githubusercontent.com/101580/167613603-a9b84d4b-6dd6-4261-989c-f2b850d00d91.png)

**Comment by 04RR** [comment]
Hey, thank you for pointing this out I will try to add that to make the implementation follow the paper as closely as possible. However this is not the official implementation, and I have made this out of my own interest.

**Comment by Kaushalya** [comment]
Thanks a lot. Sorry, I had the impression this is an official implementation.
